src/core/server/logging/logging_system.ts

```diff
@@ -67,9 +71,12 @@ export class LoggingSystem implements LoggerFactory {
    * Updates all current active loggers with the new config values.
    * @param rawConfig New config instance.
    */
-  public upgrade(rawConfig: LoggingConfigType) {
-    const config = new LoggingConfig(rawConfig)!;
-    this.applyBaseConfig(config);
+  public async upgrade(rawConfig: LoggingConfigType) {
+    // We only want the console appender for the CLI process,
+    // so we use the 'default' configuration as defined by the schema.
+    const usedConfig = this.env.isDevCliParent ? loggingConfig.schema.validate({}) : rawConfig;
+    const config = new LoggingConfig(usedConfig)!;
+    await this.applyBaseConfig(config);
```

Comment:
During my testing, I added a size-based rolling-file appender to the `root` logger. That caused races issues as both the `CLI` and the `Kibana` processes were trying to roll the file at the same time when the limit was reached.

After discussion with @spalger, we decided that the CLI should not use the logging configuration, and just fallback to the default (which is just outputting to the console)

The multi-process rolling synchronisation problematic should be handled separately from this PR, probably as a part of https://github.com/elastic/kibana/issues/68626.